packages/editor/src/components/media-placeholder/index.js

```diff
@@ -127,7 +127,8 @@ class MediaPlaceholder extends Component {
 					{ __( 'Upload' ) }
 				</FormFileUpload>
 				<MediaUpload
-					gallery={ multiple }
+					gallery={ multiple && type === "image" }
+					playlist={ multiple && type === "audio" }
```

Comment:
I don't think we support playlists yet in `MediaUpload` and even if we did, there are playlists of videos.

I think it's probably better to remove until #9169

Reply:
I hear what you are saying regarding the video playlists and that could be easily changed to something accommodating for both audio and video types.

But I have to disagree regarding the possibility to wait until #9169: analyzing that code – particularly [this part of the edit.js file](https://github.com/WordPress/gutenberg/pull/9169/files#diff-7776436ea4fc43d5443c66f7d10ca453R102) – it is clear that it simply tried to circumvent the current MediaPlaceholder limitation, recreating the whole component inside the edit.js file, only because the current MediaPlaceholder implementation does not allow to manage media types other than images.

So, I believe it is important to fix the MediaPlaceholder before refactoring the playlist block, which will allow a more streamlined playlist component as well.

Reply:
The underlying `MediaUpload` component don't support the `playlist` yet. So why to pass a useless prop?

Reply:
You are right.
I edited the proposed change accordingly.

My point is the `MediaPlaceholder` should be fixed before refactoring the playlist because of the reasons I explained above.